sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRResource.cs

```diff
@@ -119,15 +118,6 @@ public SignalRResource(string id = default(string), string name = default(string
         [JsonProperty(PropertyName = "sku")]
         public ResourceSku Sku { get; set; }
 
-        /// <summary>
-        /// Gets or sets prefix for the hostName of the SignalR service.
-        /// Retained for future use.
-        /// The hostname will be of format:
-        /// &amp;amp;lt;hostNamePrefix&amp;amp;gt;.service.signalr.net.
-        /// </summary>
-        [JsonProperty(PropertyName = "properties.hostNamePrefix")]
-        public string HostNamePrefix { get; set; }
-
```

Comment:
same comment - breaking change

Reply:
Also covered, since this property is inherited from CreateOrUpdateProperties in swagger